astropy/nddata/tests/test_ccddata.py

```diff
@@ -12,6 +12,7 @@
     InverseVariance)
 from astropy import units as u
 from astropy import log
+from astropy.io import fits
```

Comment:
Is this import necessary?

Reply:
Good catch, it is not. Was going to use it to generate the file I needed for testing, then realized I could do it another way.